DiamondLightSource/mx-bluesky
# Fix flaky backlight test

Fixes #653

Rather than check that the backlight has finished moving now we just check that we told it to move, which should be a lot less flaky.

To test:
* Run the unit tests all through a few times and confirm they do not fail with the backlight timeout error
